test/coverage/gen_build.sh

```diff
@@ -10,20 +10,26 @@ set -e
 
 [ -z "${BAZEL_BIN}" ] && BAZEL_BIN=bazel
 [ -z "${BUILDIFIER_BIN}" ] && BUILDIFIER_BIN=buildifier
+[ -z "${BUILD_PATH}" ] && BUILD_PATH="$(dirname "$0")"/BUILD
+[ -z "${REPOSITORY}" ] && REPOSITORY=""
```

Comment:
Maybe add some comments explaining what each of these is supposed to do. It was my fault  they were originally so opaque.